python/analysis/network/qgsgraphdirector.sip

```diff
@@ -1,51 +1,72 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/analysis/network/qgsgraphdirector.h                              *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
+
 %ModuleHeaderCode
 #include <qgsvectorlayerdirector.h>
 %End
 
-/**
- * \ingroup analysis
- * \class QgsGraphDirector
- * \brief Determine making the graph. QgsGraphBuilder and QgsGraphDirector
- * are implemented using "builder" design patter.
- */
 class QgsGraphDirector : QObject
 {
+%Docstring
+ Determine making the graph. QgsGraphBuilder and QgsGraphDirector implemented
+ using "builder" design patter.
+%End
+
 %TypeHeaderCode
-#include <qgsgraphbuilder.h>
+#include "qgsgraphdirector.h"
 %End
 
 %ConvertToSubClassCode
-  if ( dynamic_cast< QgsVectorLayerDirector* > ( sipCpp ) != NULL )
-    sipType = sipType_QgsVectorLayerDirector;
-  else
-    sipType = NULL;
+    if ( dynamic_cast< QgsVectorLayerDirector * >( sipCpp ) != NULL )
+      sipType = sipType_QgsVectorLayerDirector;
+    else
+      sipType = NULL;
 %End
-
-
-  signals:
-    void buildProgress( int, int ) const;
-    void buildMessage( const QString& ) const;
```

Comment:
OK, there was an error because signals were added before the public section (issue in the script). I will remove the comments, let the signal go in the sip file and merge.